File: public/js/8ball.js
$(document).ready( function() {
	$(".submit").on('click', function(e) {
		e.preventDefault();
		$.ajax('/', { 
			method:'post',
			success: function(result){ 
				$('.question span').html(result.ask);
				$('.answer span').html('"' + result.answer + '"').removeClass().addClass(result.name);
				$('p').show(); 
				},
				data: { ask: $('.ask').val() },
				dataType: 'json'
				})
		});
	});

File: 8ball.rb
require 'sinatra'
require 'sinatra/json'
require './question'

get '/' do
	erb :index
end

post '/' do
	ask = Question.new(params[:ask])
	json :ask => ask.question_empty, :answer => ask.answer_text, :name => ask.answer_class 
end